### ISSUE TITLE
Invalid token format; expecting 2 parts separated by

### ISSUE THREAD

**Issue by 499909744** [hyperledger/fabric-ca#270]
/api/v1/certificates?id=&aki=&serial=&revoked_start=&revoked_end=&expired_start=&expired_end=&notexpired=&notrevoked=&ca= 401 25 "Invalid token in authorization header: Invalid token format; expecting 2 parts separated by '.'"

**Comment by Vishva066** [comment]
I am also facing this same issue. This API request needs an enrollment token which needs to passed as an bearer token with base64 of enrollment cert. signature of the certificate and body. But there is no documentation/eg about how to generate this and what is this signature. Need help on this. If anyone has solved this kindly help !!!

**Comment by Pavani-2002** [comment]
@Vishva066  Did you figure out how to solve it?

**Comment by Vishva066** [comment]
Not yet still getting the same error

**Comment by Pavani-2002** [comment]
Yes, it is working for register API. Can you please tell me how to generate certificate_Request  (one of the body param) for /api/v1/enroll API
